thriftserver/thriftserver/base/thriftserver-db-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: thriftserver-db-pvc
+  annotations:
+    volume.beta.kubernetes.io/storage-class: "$(storage_class)"
```

Comment:
I have described the problem with this in the #227

Reply:
Moved to overlay